### PR TITLE
Last touches before v0.9.2 (kubecon)

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
## Description (git log)

libbpf was mistakenly moved to another version during a rushed release.

## Type of change

- [ ] Bug fix (non-breaking change fixing an issue, preferable).
- [x] Quick fix (minor non-breaking change requiring no issue, use with care)
- [ ] Code refactor (code improvement and/or code removal)
- [ ] New feature (non-breaking change adding functionality).
- [ ] Breaking change (cause existing functionality not to work as expected).

### CODE REVIEW

File: pkg/capabilities/capabilities.go [hunk 1]
```diff
@@ -11,7 +11,8 @@ import (
 	"kernel.org/pub/linux/libs/security/libcap/cap"
 )
 
-var Caps Capabilities // singleton for all packages
+var once sync.Once
+var Caps *Capabilities // singleton for all packages
```
Comment (yanivagman) [inline pkg/capabilities/capabilities.go:15]: Let's keep Caps unexported and rename NewCapabilities to GetInstance as we already discussed

**Review by yanivagman — COMMENTED** [review]
I think that GetInstance should not return an error, only the instance (if not initialized, panic). Then you will be able to remove all the error checking, and use the instance on the same line you get it. The only change required for this is to have an Initialize() function that will be called once (which internally calls sync.Once)

**Comment by rafaeldtinoco** [comment]
> I think that GetInstance should not return an error, only the instance (if not initialized, panic). Then you will be able to remove all the error checking, and use the instance on the same line you get it. The only change required for this is to have an Initialize() function that will be called once (which internally calls sync.Once)

I thought about this as well, I'm glad I said it. Will make the change soon and test things over for the release.

File: cmd/tracee-ebpf/flags/capabilities.go [hunk 2]
```diff
@@ -26,19 +26,15 @@ Available capabilities:
...
 			b := strings.TrimPrefix(slice, "bypass=")
-			if b == "1" || b == "true" {
-				capsConfig.BypassCaps = true
-				capsConfig.AddCaps = nil
-				capsConfig.DropCaps = nil
-				return capsConfig, nil
-			}
-			if b != "0" && b != "false" {
+			if b == "0" || b == "false" {
+				capsConfig.BypassCaps = false
```
Comment (geyslan) [inline cmd/tracee-ebpf/flags/capabilities.go:36]: Would be safe to break here?

File: pkg/capabilities/capabilities.go [hunk 3]
```diff
@@ -31,26 +32,35 @@ const (
...
-	Caps = Capabilities{}
-	return Caps.initialize(bypass)
+func Initialize(bypass bool) error {
+	var err error
+
+	once.Do(func() {
+		caps = &Capabilities{}
+		err = caps.initialize(bypass)
+	})
+	if err != nil {
```
Comment (yanivagman) [inline pkg/capabilities/capabilities.go:48]: I think you can simply return err at that point

**Review by yanivagman — APPROVED** [review]
LGTM!

**Pull request merged**